site/ja/guide/tensor.ipynb

```diff
@@ -47,10 +47,10 @@
       },
       "source": [
         "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
-        "  <td><a target=\"_blank\" href=\"https://www.tensorflow.org/guide/tensor\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\"> TensorFlow.orgで表示</a></td>\n",
+        "  <td>     <a target=\"_blank\" href=\"https://www.tensorflow.org/guide/tensor\" class=\"\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\">TensorFlow.org で表示</a>   </td>\n",
         "  <td><a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs-l10n/blob/master/site/ja/guide/tensor.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\"> Google Colab で実行</a></td>\n",
-        "  <td><a target=\"_blank\" href=\"https://github.com/tensorflow/docs-l10n/blob/master/site/ja/guide/tensor.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\">GitHub でソースを表示{</a></td>\n",
-        "  <td><a href=\"https://storage.googleapis.com/tensorflow_docs/docs-l10n/site/ja/guide/tensor.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\">ノートブックをダウンロード/a0}</a></td>\n",
+        "  <td>     <a target=\"_blank\" href=\"https://github.com/tensorflow/docs-l10n/blob/master/site/ja/guide/tensor.ipynb\" class=\"\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\">GitHub でソースを表示</a>   </td>\n",
```

Comment:
I think the `class=\"\"` is being added by GitLocalize since I don't see it in the [source notebook](https://raw.githubusercontent.com/tensorflow/docs-l10n/master/site/en-snapshot/guide/tensor.ipynb).

This shouldn't hurt anything but just want to confirm the behavior is intentional. @ilyaspiridonov

Reply:
@ilyaspiridonov I haven't seen empty attributes in PRs created by GitLocalize yet. Forwarded this to the development team.